src/lightkurve/interact.py

```diff
@@ -385,291 +407,191 @@ def make_lightcurve_figure_elements(lc, lc_source, ylim_func=None):
+               f"ra: {getattr(tpf, 'ra', None)}, dec: {getattr(tpf, 'dec', None)}")
         raise LightkurveError(msg) from err
 
-    # Use pixel scale for query size
-    pix_scale = 4.0  # arcseconds / pixel for Kepler, default
-    if tpf.mission == "TESS":
-        pix_scale = 21.0
-    # We are querying with a diameter as the radius, overfilling by 2x.
-    from astroquery.vizier import Vizier
+    provider_kwargs["coord"] = c1
 
-    Vizier.ROW_LIMIT = -1
-    with warnings.catch_warnings():
-        # suppress useless warning to workaround  https://github.com/astropy/astroquery/issues/2352
-        warnings.filterwarnings(
-            "ignore", category=u.UnitsWarning, message="Unit 'e' not supported by the VOUnit standard"
-        )
-        result = Vizier.query_region(
-            c1,
-            catalog=["I/345/gaia2"],
-            radius=Angle(np.max(tpf.shape[1:]) * pix_scale, "arcsec"),
-        )
-    no_targets_found_message = ValueError(
-        "Either no sources were found in the query region " "or Vizier is unavailable"
-    )
-    too_few_found_message = ValueError(
-        "No sources found brighter than {:0.1f}".format(magnitude_limit)
-    )
-    if result is None:
-        raise no_targets_found_message
-    elif len(result) == 0:
-        raise too_few_found_message
-    result = result["I/345/gaia2"].to_pandas()
-    result = result[result.Gmag < magnitude_limit]
-    if len(result) == 0:
-        raise no_targets_found_message
-    # drop all the filtered rows, it makes subsequent TESS-specific processing easier (to add rows/columns)
-    result.reset_index(drop=True, inplace=True)
-    result['magForSize'] = result['Gmag']  # to be used as the basis for sizing the dots in plots
-    return result
-
-
-def add_gaia_figure_elements(tpf, fig, magnitude_limit=18):
-    """Make the Gaia Figure Elements"""
-
-    result = _get_nearby_gaia_objects(tpf, magnitude_limit)
-
-    source_colnames_extras = []
-    tooltips_extras = []
-    try:
-        result, source_colnames_extras, tooltips_extras = _add_nearby_tics_if_tess(tpf, magnitude_limit, result)
-    except Exception as err:
-        warnings.warn(
-            f"interact_sky() - cannot obtain nearby TICs. Skip it. The error: {err}",
-            LightkurveWarning,
-        )
+    if provider_kwargs.get("radius") is None:
+        # use the default search radius, scaled to the TargePixelFile size
 
-    ra_corrected, dec_corrected, _ = _correct_with_proper_motion(
-            np.nan_to_num(np.asarray(result.RA_ICRS)) * u.deg, np.nan_to_num(np.asarray(result.DE_ICRS)) * u.deg,
-            np.nan_to_num(np.asarray(result.pmRA)) * u.milliarcsecond / u.year,
-            np.nan_to_num(np.asarray(result.pmDE)) * u.milliarcsecond / u.year,
-            Time(2457206.375, format="jd", scale="tdb"),
+        # Use pixel scale for query size
+        pix_scale = 4.0  # arcseconds / pixel for Kepler, default
```

Comment:
Explicit if statements are now used to explicitly check for TESS / K2 / Kepler. 
For now, it will throw a `ValueError` if the tpf is not from any of the 3 missions.

Alternatively, we could issue a warning and use a default pixel scale. Maybe 21" as it'd be a larger search radius (than 4") ?